src/main/resources/db/migration/V0__drop_and_create_schema.sql

```diff
@@ -0,0 +1,7 @@
+# -- V001__drop_and_create_schema.sql
+#
+# DROP SCHEMA IF EXISTS `kniznica_v5`;
+#
+# CREATE SCHEMA `kniznica_v5`;
+#
+# USE `kniznica_v5`;
```

Comment:
Why it is commented?

Reply:
wanted to delete and recreate the schema at startup

Reply:
if you dont need it please delete it.